packages/itwinui-css/src/tree/tree.scss

```diff
@@ -2,19 +2,22 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-$iui-active-outline: thin solid var(--iui-color-border-accent);
-
-// padding of small borderless button + 1px transparent border
-$iui-expander-inline-padding: calc(var(--iui-size-2xs) + 1px);
-
-// width of small borderless button (button is 1:1 ratio)
-$iui-expander-button-width: calc(var(--iui-component-height-small) + 2px);
+$iui-active-outline: 1px solid var(--iui-color-border-accent);
 
 .iui-tree {
   @include mixins.iui-reset;
+  --_iui-expander-button-width: calc(var(--iui-component-height-small) + 2px);
```

Comment:
this can be moved back to sass variable as it's not changed anywhere

Reply:
Can do, reverted back to SASS var.